workspace/utilities/admin-tools/admin-tools.css

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+.admin-tools{
+	position: fixed;
+	top: 10px;
+	left: 10px;
+	background-color: #2f77eb;
+	color: white;
+	font-size: 14px;
+	font-family: 'Roboto', sans-serif;
+	z-index: 99999;
+}
+
+.admin-tools a{
+	color: white;
+	text-decoration: none;
+}
+
+.admin-tools .notifier{
+	position: fixed;
+	top: 10px;
+	left: 10px;
+	width: 10px;
+	height: 10px;
+	margin: 5px 0 0 7px;
+	border-radius: 50%;
+	z-index: 99999;
+	box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+}
+
+.admin-tools .widgets{
+	position: absolute;
+	top:0;
+	opacity: 0;
+	display: none;
+	background-color: #1976D2;
+	border-radius: 8px;
+	box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+}
+
+.admin-tools:hover .widgets{
+	display: block;
+	animation: admin-tools-fadein 0.2s forwards;
```

Comment:
Why using an animation here? A transition seems to be the better option since it supports both direction (hover in and out) while the animation will reset abrutly on hover out.

Reply:
;)